Form/Type/ApiCommentType.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\NewsBundle\Form\Type;
+
+use Sonata\CoreBundle\Form\Type\BaseDoctrineORMSerializationType;
+
+class ApiCommentType extends BaseDoctrineORMSerializationType
```

Comment:
> Arf, should I add my name on 90% of the sonata code base ?

You could, if you want :trollface: But we wanna add the tag to all new classes, just to have general workflow.